webrtc.html

```diff
@@ -1882,9 +1882,8 @@ <h4>Set the RTCSessionDescription</h4>
                             </ol>
                           </li>
                           <li>
-                            <p>If the <a>media description</a> is rejected, and
-                            <var>transceiver</var> is not already stopped, <a>stop
-                            the RTCRtpTransceiver</a> <var>transceiver</var>.</p>
+                            <p>If the <a>media description</a> is rejected,
+                            <a>stop the RTCRtpTransceiver</a> <var>transceiver</var>.</p>
```

Comment:
I claim we still need to check for already stopped to handle rollback for instance.